cmd/util/ledger/migrations/staged_contracts_migration.go

```diff
@@ -148,11 +153,131 @@ func (m *StagedContractsMigration) InitMigration(
 	return nil
 }
 
+// collectAndRegisterStagedContractsFromPayloads scan through the payloads and collects the contracts
+// staged through the `MigrationContractStaging` contract.
+func (m *StagedContractsMigration) collectAndRegisterStagedContractsFromPayloads(allPayloads []*ledger.Payload) error {
+	var stagingAccount string
+
+	// If the contracts are already passed as an input to the migration
+	// then no need to scan the storage.
+	// TODO: Maybe do this anyway?
+	if len(m.contractsByLocation) > 0 {
+		return nil
+	}
+
+	switch m.chainID {
+	case flow.Testnet:
+		stagingAccount = "0x2ceae959ed1a7e7a"
+	case flow.Mainnet:
+		// TODO:
+		stagingAccount = "0x2ceae959ed1a7e7a"
```

Comment:
@sisyphusSmiling We need the address where [MigrationContractStaging](https://contractbrowser.com/A.2ceae959ed1a7e7a.MigrationContractStaging) is deployed in different networks. Could you please double check if the address is correct for `testnet`?
Also, do you know what the address should be for `mainnet`?

Reply:
Contract looks correct for Testnet and added a suggestion with the Mainnet address

Reply:
Thank you!